source/Tests/Iciclecreek.Bot.Builder.Dialogs.Recognizers.Tests/Startup.cs

```diff
@@ -39,6 +41,8 @@ public static void Initialize(TestContext testContext)
             ComponentRegistration.Add(new AdaptiveTestingComponentRegistration());
             ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<QLuceneBotComponent>());
             ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<LucyBotComponent>());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<LuceneBotComponent>());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<LuisBotComponent>());
```

Comment:
Their is a new way of registering components that does not require explicit registration.  We should switch to BotComponent something like this:
    public class LuisBotComponent : BotComponent
    {
        /// <inheritdoc/>
        public override void ConfigureServices(IServiceCollection services, IConfiguration configuration)
        {
            // Converters
            services.AddSingleton<JsonConverterFactory, JsonConverterFactory<ArrayExpressionConverter<DynamicList>>>();
            services.AddSingleton<JsonConverterFactory, JsonConverterFactory<ArrayExpressionConverter<ExternalEntity>>>();

            // Declarative types
            services.AddSingleton<DeclarativeType>(sp => new DeclarativeType<LuisAdaptiveRecognizer>(LuisAdaptiveRecognizer.Kind));
        }
    }